src/dlgTriggerEditor.cpp

```diff
@@ -665,19 +665,15 @@ void dlgTriggerEditor::closeEvent(QCloseEvent *event)
 
 void dlgTriggerEditor::readSettings()
 {
-    QSettings settings("mudlet", "Mudlet");
-
     /*In case sensitive environments, two different config directories 
     were used: "Mudlet" for QSettings, and "mudlet" anywhere else.
     Furthermore, we skip the version from the application name to follow the convention.
     For compatibility with older settings, if no config is loaded 
     from the config directory "mudlet", application "Mudlet", we try to load from the config 
     directory "Mudlet", application "Mudlet 1.0". */
-    if(settings.value("pos") == 0)
-    {
-        QSettings settings("Mudlet","Mudlet 1.0");
-    }
-
+    QSettings settings_new("mudlet","Mudlet");
+    QSettings settings((settings_new.value("pos")==0? "Mudlet":"mudlet"),(settings_new.value("pos")==0? "Mudlet 1.0":"Mudlet"));
```

Comment:
I think you should be using `QSettings::contains("pos")` rather than testing for the (unspecified here but nevertheless used) default null `QVarient` returned from `QSettings::value( const QString &key, const QVariant &defaultValue = QVariant() )`...